ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.model.js

```diff
@@ -28,13 +29,20 @@ module.exports = Backbone.AssociatedModel.extend({
         mouseLon: undefined,
         clickLat: undefined,
         clickLon: undefined,
+        dms: undefined,
         target: undefined,
         targetMetacard: undefined
     },
     updateClickCoordinates: function () {
+
+        var lat = this.get('mouseLat');
+        var lon = this.get('mouseLon');
+        var dms = ol.coordinate.toStringHDMS([lon, lat]);
```

Comment:
Do you know if we want any more precision on the DMS values? Precision can be lengthened by passing integer to `toStringHDMS`

Reply:
I matched the values I was seeing in the bottom left corner (minus the ZERO-padding).